src/main/webapp/app/course/manage/overview/course-management-card.component.ts

```diff
@@ -51,7 +51,6 @@ export class CourseManagementCardComponent implements OnInit {
 
     getStatisticForExercise(exercise: Exercise) {
         const x = this.statistics[exercise.id!];
-        console.log(x);
         return x;
```

Comment:
`return this.statistics[exercise.id!];`

Reply:
👍

Reply:
c4f74cf093cfd8e2afd0455b877365689c3d8834